### ISSUE TITLE
Error when visiting a post from a newly created 'post type'.

### ISSUE THREAD

**Issue by ThinkGraphical** [anomalylabs/posts-module#25]
Steps to reproduce:

1. Create a new post type, assign fields,...
2. Create a new post with said post type
3. Visit the post in the front

Error being thrown:

`ErrorException in post.twig line 1:
An exception has been thrown during the rendering of a template ("Illegal offset type in isset or empty") in "/home/projects/apps/pyrocms-poc/core/anomaly/posts-module/resources/views/posts/post.twig" at line 1.`

**Comment by RyanThompson** [comment]
You on windows?

**Comment by ThinkGraphical** [comment]
No, mac & yes fresh install.

**Comment by RyanThompson** [comment]
Can you try adding `.value` to this line: https://github.com/anomalylabs/posts-module/blob/master/resources/views/posts/post.twig#L1 <https://github.com/anomalylabs/posts-module/blob/master/resources/views/posts/post.twig#L1>

**Comment by theshaunwalker** [comment]
tl;dr; `post.type.theme_layout.key` works

Adding `.value` gets the form input value, not the layout path itself.

i.e. two layouts
```
+ default.twig
+ posts
    + default.twig
```
The form inputs in the dropdown look like `Default` and `Posts > Default` and this is whats output via `post.type.theme_layout.value`

So `.key` needs appended onto the @RyanThompson 's quoted line

**Comment by theshaunwalker** [comment]
Made PR for this change https://github.com/anomalylabs/posts-module/pull/26

**Comment by RyanThompson** [comment]
Well done @theshaunwalker thank you!
